.travis.yml

```diff
@@ -85,6 +85,7 @@ install:
   rm -rf "$RUSTUP_HOME"toolchains/*/etc
   rm -rf "$RUSTUP_HOME"toolchains/*/share
 before_script:
+- rustup component add clippy-preview
```

Comment:
I think this needs to happen in the install step, before "   # Remove unnnecessary cargo and rustup directories."

Reply:
e.g. https://github.com/hayd/deno/commit/d5d40e846d2796b2f1b3251ac2454784df7886ce

Reply:
I'm closing this up since I'm not making any reasonable progress 😞